spec/requests/api/orchestration_template_spec.rb

```diff
@@ -125,4 +125,70 @@
       expect(OrchestrationTemplate.exists?(hot.id)).to be_falsey
     end
   end
+
+  context 'orchestration template copy' do
+    it 'copy template with the same content' do
+      test_ot_1 = FactoryGirl.create(:orchestration_template_cfn)
+
+      api_basic_authorize collection_action_identifier(:orchestration_templates, :copy)
```

Comment:
To be consistent with spacing can you try to group lines that belong to each phase of the test, i.e. all setup steps together, all execution steps, all verification steps, etc...